crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -181,6 +186,31 @@ impl SystemExecutor for MultiThreadedExecutor {
             }
         }
 
+        // If stepping is enabled, make sure we skip those systems that should
+        // not be run.
+        #[cfg(feature = "bevy_debug_stepping")]
```

Comment:
Question: Why do we need to limit this block to just this feature if `skip_systems` is available as a parameter outside of it? Same for these other files.

Reply:
The feature flag was requested as a mechanism to eliminate all performance impacts of stepping.  I may have gone overboard here, but as there's no other feature or functionality that is currently using the `skip_systems` argument, this felt like the best approach to meet that goal of no performance impact.

We can always remove this cfg check if another feature wants to start leveraging skip lists.